pinc/graph_data.inc

```diff
@@ -101,11 +100,14 @@ function user_logging_on($past, $preceding)
     ///////////////////////////////////////////////////
     //query db and put results into arrays
 
-    $result = mysqli_query(DPDatabase::get_connection(), "
-      SELECT DATE_FORMAT(FROM_UNIXTIME(time_stamp),'$date_format'), $column_name
+    $sql = sprintf("SELECT DATE_FORMAT(FROM_UNIXTIME(time_stamp),'%s'), %s
```

Comment:
Column names can't be escaped this way. Please leave `$column_name` inline in the SQL per the [best practices](https://www.pgdp.net/wiki/DP_Code_Best_Practices#Column_names_as_variables).

Reply:
oh yes, thanks for reminding me.